content/20.treatments.md

```diff
@@ -226,6 +226,16 @@ In principle, drugs that reduce the expression of these proteins or sterically h
 
 A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
 
+Angiotensin-converting enzyme inhibitors (ACEis) and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications, prescribed well over 100,000,000 times annually in the United States, for example.
+Data from some animal models suggest several but not all ACEis and several ARBs increase ACE2 expression in cells of some organs.
```

Comment:
Refs?

Reply:
This reference contains a very comprehensive table addressing the effect of ACE inhibitors and ARBs on ACE2 in cells in various organs in different animal models: @doi:10.1093/cvr/cvaa097. We also have a somewhat less comprehensive discussion on our blog post tracking this issue: @url:http://www.nephjc.com/news/covidace2, but the publication is probably a better citation.